src/main/java/org/commonjava/util/sidecar/services/ReportService.java

```diff
@@ -31,67 +34,107 @@
 import javax.annotation.PostConstruct;
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.HashMap;
 
 import static org.commonjava.util.sidecar.services.PreSeedConstants.FOLO_BUILD;
+import static org.commonjava.util.sidecar.services.PreSeedConstants.STARTUP_INIT;
 import static org.commonjava.util.sidecar.util.SidecarUtils.getBuildConfigId;
 
-@RegisterForReflection
 @ApplicationScoped
 public class ReportService
 {
     private final Logger logger = LoggerFactory.getLogger( getClass() );
 
-    private TrackedContent trackedContent;
+    public TrackedContent trackedContent;
+
+    private HashMap<String,HistoricalEntryDTO> historicalContentMap;
 
     @Inject
     ObjectMapper objectMapper;
 
+    @Inject
+    Classifier classifier;
+
+    @Inject
+    ProxyService proxyService;
+
     @PostConstruct
     void init(){
         this.trackedContent = new TrackedContent();
+        this.historicalContentMap = new HashMap<>();
```

Comment:
Actually we can move these to fields construction instead of a PostConstruct method if it only contains fields initialization. And if these fields are immutable, set them to final is better.